### PR TITLE
Releasing 2.11.0

### PR DESCRIPTION
**Pull request by smahima27**

### CODE REVIEW

**Review by joshcooper — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file.
 
+## [2.10.1](https://github.com/voxpupuli/beaker-hostgenerator/tree/2.10.1) (2024-04-08)
+
+[Full Changelog](https://github.com/voxpupuli/beaker-hostgenerator/compare/2.10.0...2.10.1)
+
+**Implemented enhancements:**
+
+- Add beaker-hostgenerator support for Fedora40-x86\_64 [\#361](https://github.com/voxpupuli/beaker-hostgenerator/pull/361) ([smahima27](https://github.com/smahima27))
```
Comment (bastelfreak) [inline CHANGELOG.md:11]: This is an enhancement, so please make this a 2.11.0 release. patch releases only contain bugfixes.

Comment (smahima27) [reply CHANGELOG.md]: Fixed version number

**Review by bastelfreak — APPROVED** [review]

**Comment by smahima27** [comment]
@bastelfreak can you please merge it.

**Comment by bastelfreak** [comment]
merged and pushed the release

**Pull request merged**